json-lint/utils/schemaUtils.js

```diff
@@ -26,7 +26,7 @@ const getErrorMessage = (command, jsonPath, status, execPath) => `Error merging
 function getMarfeelExtendedJson(jsonPath, command = JSON_MERGE_COMMAND ) {
 	const mrfJson = spawnSync(command , [ jsonPath ]);
 	
-	if(!!mrfJson.status){
+	if(mrfJson.stderr.length > 0 ||!!mrfJson.status){
```

Comment:
IMHO if mrf-json prints to stderr and return status 0 then we should fix mrf-json

Reply:
the problem is that returns a "null" in stderror and we had a concatenation of issues:
!"null" = false 
 JSON.parse("null") ==> valid